sources/SEDIT-TOPLEVEL

```diff
@@ -231,8 +265,7 @@
 
               (IL:* IL:|;;| "found a matching context elsewhere")
 
-              (IL:RESETFORM (IL:EDITMODE SEDIT)
-                     (IL:EDITDEF NAME TYPE NIL NIL '(:DONTWAIT))))))))
+              (IL:EDITDEF NAME TYPE NIL NIL '(:DONTWAIT)))))))
 
```

Comment:
it looks like this PR loses the edit I made in a previous PR

Reply:
this is screwy -- I updated rmk6-redux to have the right EDITMODE but it should have been this one rmk7 instead.
once more? 

i got an error that I haven't tracked down -- too many arguments (2) to sedit::start-process (which only takes one).